python/lsst/ts/mtaos/mtaos.py

```diff
@@ -505,27 +539,44 @@ async def do_runWEP(self, data):
             result="runWEP started.",
         )
 
+        await self._execute_wavefront_estimation(data)
+
+    async def _execute_wavefront_estimation(self, data):
```

Comment:
Instead of passing `data`, expand the input parameters to explicitly pass what you need here, e.g.:

```
async def _execute_wavefront_estimation(self, visit_id, extra_id, use_ocps, timestamp, identity,  config):
    ...
```

These are the parameters I could identify that are used from `data`.